pkg/driver/screen/x11_linux.go

```diff
@@ -59,11 +59,11 @@ func (s *screen) Close() error {
 	return nil
 }
 
-func (s *screen) VideoRecord(p prop.Media) (video.Reader, error) {
-	if p.FrameRate == 0 {
-		p.FrameRate = 10
+func (s *screen) VideoRecord(p, req prop.Media) (video.Reader, error) {
+	if req.FrameRate == 0 {
```

Comment:
`p` is a prop selected from the list returned by `Properties()`, so it can't contain user requested values for continuously settable parameters.